README.md

```diff
@@ -656,6 +679,7 @@ If you would like to become an active contributor to this project please follow
 
 | Version           | SHA1                                                                                      | Remarks                                               |
 |-------------------|-------------------------------------------------------------------------------------------|-------------------------------------------------------|
+| 1.6.0       | [1.6.0](https://github.com/Azure/azure-libraries-for-java/tree/v1.6.0)               | Tagged release for 1.6.0 version of Azure management libraries |
```

Comment:
Missing an entry for 1.5.1

```
| 1.5.1       | [1.5.1](https://github.com/Azure/azure-libraries-for-java/tree/v1.5.1)               | Tagged release for 1.5.1 version of Azure management libraries | 
```

Reply:
Replaced the 1.5.0 entry with 1.5.1 which is the correct entry